docs/modules/tutorials/pages/tech-basics-for-newbies.adoc

```diff
@@ -0,0 +1,96 @@
+= Trainings
+:noindex:
+== VIVIDUS usage related technical basics
```

Comment:
 From my point of view that's a lot to take in before VIVIDUS learning starts. There are tons of generic topics, from my point of view it makes sense to move them into module preconditions, e.g. before the web module it would be nice to read about locators.